packages/aragon-cli/src/commands/apm_cmds/util/preprare-files.js

```diff
@@ -25,21 +25,15 @@ async function prepareFilesForPublishing(
   const filter = ignore().add(ignorePatterns)
   const projectRoot = findProjectRoot()
 
-  function createFilter(files, ignorePath) {
-    let f = fs.readFileSync(ignorePath).toString()
-    files.forEach(file => {
-      f = f.concat(`\n!${file}`)
-    })
-    return f
-  }
-
   const ipfsignorePath = path.resolve(projectRoot, '.ipfsignore')
   if (pathExistsSync(ipfsignorePath)) {
-    filter.add(createFilter(files, ipfsignorePath))
+    const ipfsignoreFile = fs.readFileSync(ipfsignorePath).toString()
+    filter.add(ipfsignoreFile)
   } else {
     const gitignorePath = path.resolve(projectRoot, '.gitignore')
     if (pathExistsSync(gitignorePath)) {
-      filter.add(createFilter(files, gitignorePath))
+      const gitignoreFile = fs.readFileSync(gitignorePath).toString()
+      filter.add(gitignoreFile)
     }
   }
 
```

Comment:
It seems to work fine without, but [`ignore`'s pathname conventions](https://www.npmjs.com/package/ignore#pathname-conventions) says we should strip out the `.` and use `path.join()` in `filterIgnoredFiles()` 🤷‍♂